PyInstaller/utils/hooks/__init__.py

```diff
@@ -1,35 +1,23 @@
-#-----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
```

Comment:
Sorry I have a branch that changes all of these and this one must have leaked over. I'll change it back.